src/module.ts

```diff
@@ -29,7 +29,7 @@ export default defineNuxtModule<ModuleOptions>({
     usePlugin: true, // legacy opt. for enableSudoMode
     bridge: true,
     devtools: false,
-    globalDir: '~/storyblok',
+    componentsDir: '~/storyblok',
```

Comment:
@Dawntraoz I think we should offer an alias with a deprecation note for the previous `globalDir` otherwise this should be treated as a breaking change.

Reply:
Hi @alvarosabu as discussed already with Alex on Slack, only Markus was using it, and it wasn't document it, so the idea is to add this one to the README and make it official! He's already informed of the change 🫂